docs/source/exporters/tflite/overview.mdx

```diff
@@ -0,0 +1,36 @@
+<!--Copyright 2023 The HuggingFace Team. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+the License. You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+specific language governing permissions and limitations under the License.
+-->
+
+# Overview
+
+🤗 Optimum handles the export of PyTorch or TensorFlow models to TFLite in the `exporters.tflite` module. In case a PyTorch model is passed, it will automatically be converted to TensorFlow by transformers' [TFPreTrainedModel.from_pretrained()](https://huggingface.co/docs/transformers/main/en/main_classes/model#transformers.TFPreTrainedModel.from_pretrained) before the TFLite export.
```

Comment:
No just TensorFlow models. You have some models in PyTorch that do not have a TensorFlow implementation.
I would distinguish between weights on the Hub, and what we call "model", since here we only support TensorFlow model implementations.